vizro-core/src/vizro/static/css/table.css

```diff
@@ -1,93 +1,38 @@
-#right-side .dash-cell {
-  height: 48px !important;
+#dashboard-container .dash-cell {
+  height: 3rem !important;
 }
 
-#right-side .dash-table-container {
-  padding-bottom: 12px;
-}
-
-#right-side
-  .dash-table-container
-  .dash-spreadsheet-container
-  .dash-spreadsheet-inner
-  th {
-  background-color: var(--right-side-bg);
-  color: var(--text-primary);
+#dashboard-container .dash-spreadsheet-inner th {
+  background-color: inherit;
   padding: 10px 0;
 }
 
-#right-side
-  .dash-table-container
-  .dash-spreadsheet-container
-  .dash-spreadsheet-inner
-  td {
-  background-color: var(--right-side-bg);
-  color: var(--text-primary);
-}
-
-#right-side
-  .dash-table-container
-  .dash-spreadsheet-container
-  .dash-spreadsheet-inner
-  table {
-  --hover: var(--right-side-bg);
-
-  width: 100%;
+#dashboard-container .dash-spreadsheet-inner td {
+  background-color: inherit;
 }
 
-#right-side
-  .dash-table-container
-  .dash-spreadsheet-container
-  .dash-spreadsheet-inner
-  td
-  .dash-cell-value.unfocused {
-  font-size: 14px;
+#dashboard-container .dash-spreadsheet-inner table {
+  --hover: inherit;
 }
 
-#right-side
-  .dash-table-container
-  .dash-spreadsheet-container
-  .dash-spreadsheet-inner
-  .column-header-name {
-  font-size: 14px;
+/* Styling for pagination element */
+#dashboard-container .first-page,
+#dashboard-container .previous-page,
```

Comment:
I'm assuming you are repeating `#dashboard-container` for higher specificity? I'm just curious would we achieve the same result without the repetition?

Reply:
It would be fine without it. However, if you take a look at the console in the current scratch dev example, you'll see that one dash selector actually has a higher specificity. So if we remove the `dashboard-container` id, the only reason this works is because the current dash selector does not have these properties defined. In this case, it's just safer to add it, so our selector always has higher specificity. 

![Screenshot 2024-12-17 at 13 42 08](https://github.com/user-attachments/assets/94189af1-5fad-4ae0-836d-23bdea39d1ed)